packages/@o3r/configuration/src/devkit/configuration-devtools.console.spec.ts

```diff
@@ -40,7 +40,7 @@ describe('Configuration DevTools console', () => {
             configuration: {
               ids: ['@scope/package#componentTest'],
               entities: {
-                // eslint-disable-next-line @typescript-eslint/naming-convention
+                // eslint-disable-next-line @typescript-eslint/naming-convention -- id
```

Comment:
Creating a const for the name would avoir this exception (and it is used just a bit above)

Reply:
Maybe a good example for the discussion for the different configuration between source and test :)

Reply:
Since these are few isolated cases, maybe it doesn't justify an extra config only for this.

Reply:
it seems a bit overkill to create a constant just to avoid et eslint-disable in a test

Reply:
In that case the `const` can be justify as you want to align it with the `ids` above.
But I agree it is not always the case.

This can be an ajustement of the rule for our projects but it does not justify, for me, to have this exception for all the apps